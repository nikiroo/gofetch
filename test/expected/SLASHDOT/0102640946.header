0Valve Explains How It Decides Who's a 'Straight Up Troll' Publishing Video Games On Steam (vice.com)	null/SLASHDOT/0102640946		70
i  Thursday September 06, 2018 @11:30PM (BeauHD)
i  from the behind-the-scenes dept.
i
i    An   anonymous   reader   quotes  a  report  from  Motherboard:
i    Wednesday,  Valve,  the  company  that operates the huge online
i    video  game store Steam, shared more details about how it plans
i    to  control  and  moderate  the ever-increasing number of games
i    published  on  its  platform.  In the post published Wednesday,
i    Valve  shared  more  details  about  how  it determines what it
i    considers  "outright trolling." "It is vague and we'll tell you
i    why,"  Valve  wrote.  "You're  a denizen of the internet so you
i    know  that  trolls come in all forms. On Steam, some are simply
i    trying  to rile people up with something we call 'a game shaped
i    object'  (ie: a crudely made piece of software that technically
i    and  just barely passes our bar as a functioning video game but
i    isn't  what  99.9% of folks would say is "good.") Valve goes on
i    to  explain  that  some  trolls are trying to scam folks out of
i    their  Steam  inventory items (digital items that can be traded
i    for  real  money),  while others are trying to generate a small
i    amount  of  money  through a variety of schemes that have to do
i    with  how  developers  use  keys  to  unlock Steam games, while
i    others  are  trying  to  "incite  and sow discord." "Trolls are
i    figuring  out new ways to be loathsome as we write this," Valve
i    said.  "But  the  thing these folks have in common is that they
i    aren't  actually  interested  in good faith efforts to make and
i    sell  games to you or anyone. When a developer's motives aren't
i    that,  they're  probably  a troll." One interesting observation
i    Valve   shares  in  the  blog  post  is  that  it  rarely  bans
i    individual  games  from  Steam,  and more often bans developers
i    and/or  publishers  entirely.  [...] Valve said that its review
i    process  for  determining  that something may be a "troll game"
i    is  a  "deep  assessment"  that  involves investigating who the
i    developer  is, what they've done in the past, their behavior on
i    Steam   as   a   developer,   as   a  customer,  their  banking
i    information, developers they associate with, and more.
i
