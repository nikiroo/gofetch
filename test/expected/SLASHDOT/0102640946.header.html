<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0102640946.html'>Valve Explains How It Decides Who's a 'Straight Up Troll' Publishing Video Games On Steam (vice.com)</a></h2>
	<div class='details'>(Thursday September 06, 2018 @11:30PM (BeauHD) 
from the behind-the-scenes dept.)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		An anonymous reader quotes a report from Motherboard:<br/><br/>&gt; Wednesday, Valve, the company that operates the huge online video game store Steam, shared more details about [1]how it plans to control and moderate the ever-increasing number of games published on its platform . In the post [2]published Wednesday , Valve shared more details about how it determines what it considers &quot;outright trolling.&quot; &quot;It is vague and we&#x27;ll tell you why,&quot; Valve wrote. &quot;You&#x27;re a denizen of the internet so you know that trolls come in all forms. On Steam, some are simply trying to rile people up with something we call &#x27;a game shaped object&#x27; (ie: a crudely made piece of software that technically and just barely passes our bar as a functioning video game but isn&#x27;t what 99.9% of folks would say is &quot;good.&quot;)<br/><br/>&gt;<br/><br/>&gt; Valve goes on to explain that some trolls are trying to scam folks out of their Steam inventory items (digital items that can be traded for real money), while others are trying to generate a small amount of money through a variety of schemes that have to do with how developers use keys to unlock Steam games, while others are trying to &quot;incite and sow discord.&quot; &quot;Trolls are figuring out new ways to be loathsome as we write this,&quot; Valve said. &quot;But the thing these folks have in common is that they aren&#x27;t actually interested in good faith efforts to make and sell games to you or anyone. When a developer&#x27;s motives aren&#x27;t that, they&#x27;re probably a troll.&quot; One interesting observation Valve shares in the blog post is that it rarely bans individual games from Steam, and more often bans developers and/or publishers entirely. [...] Valve said that its review process for determining that something may be a &quot;troll game&quot; is a &quot;deep assessment&quot; that involves investigating who the developer is, what they&#x27;ve done in the past, their behavior on Steam as a developer, as a customer, their banking information, developers they associate with, and more.<br/><br/><br/><br/>[1] https://motherboard.vice.com/en_us/article/9kmej7/valve-it-really-does-seem-bad-games-are-made-by-bad-people<br/><br/>[2] https://steamcommunity.com/games/593110/announcements/detail/1708442022337025126
	</div>
<hr/>
</div>
</body>
